ios/RNSentry.m

```diff
@@ -215,6 +227,7 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
 
+#if TARGET_OS_IPHONE
```

Comment:
That also works for `macCatalyst`, so I'd rather add that too.
https://github.com/getsentry/sentry-dart/blob/main/flutter/ios/Classes/SentryFlutterPluginApple.swift#L368-L383=